openfasoc/generators/glayout/glayout/flow/blocks/strongARM_US/strongARM.py

```diff
@@ -0,0 +1,160 @@
+from gdsfactory.components import rectangle
+from gdsfactory import Component
+from glayout.flow.pdk.mappedpdk import MappedPDK
+from glayout.flow.primitives.fet import nmos
+from glayout.flow.placement.two_transistor_interdigitized import two_pfet_interdigitized
+from glayout.flow.placement.four_transistor_interdigitized import generic_4T_interdigitzed
+from glayout.flow.placement.two_transistor_interdigitized import two_nfet_interdigitized
+from glayout.flow.pdk.util.comp_utils import prec_ref_center, movey, evaluate_bbox, movex, align_comp_to_port
+from glayout.flow.routing.smart_route import smart_route, c_route, straight_route, L_route
+from glayout.flow.pdk.sky130_mapped import sky130_mapped_pdk
+from glayout.flow.spice import Netlist
+
+def diffPair_netlist(fetL: Component, fetR: Component):
+    diffPair_netlist = Netlist(circuit_name ='diffPair', nodes=['VIP', 'VIM', 'VD1', 'VD2', 'VTAIL', 'VBULK'])
+    diffPair_netlist.connect_netlist(
+        fetL.info['netlist'],
+        [('G','VIP'), ('D','VD1'), ('S','VTAIL'), ('B','VBULK')]
+    )
+    diffPair_netlist.connect_netlist(
+        fetR.info['netlist'],
+        [('G','VIM'), ('D','VD2'), ('S','VTAIL'), ('B','VBULK')]
+    )
+    return diffPair_netlist
+
+def diffPair(pdk: MappedPDK, width, length):
```

Comment:
Can the existing diff pair component be used for this? Or alternatively, modify it slightly to make it more customizable to work with this?